pkg/publisher/webhookPublisher.go

```diff
@@ -1,12 +1,9 @@
 /*
 Copyright 2017 The Fission Authors.
-
```

Comment:
Before the CI pass (https://travis-ci.org/fission/fission/builds/549017388), could you restore the blank line accidentally deleted here?

Reply:
Sorry, I just saw your mesage. Restored already!